benchmarks/2023/benchmarks-schema.yaml

```diff
@@ -0,0 +1,84 @@
+$id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/benchmarks-schema.yaml
```

Comment:
This Schema file is a copy-paste from PY22. Is this what it should be?

Reply:
Yes, updates had been added there.